Libraries/Hotcakes.Commerce/BusinessRules/OrderTasks/StartPaypalExpressCheckout.cs

```diff
@@ -116,27 +123,27 @@ public override bool ProcessCheckout(OrderTaskContext context)
 							string shippingTotal = shippingTotalWithoutTax.ToString("N", CultureInfo.InvariantCulture);
 
 							string orderTotal = context.Order.TotalGrand.ToString("N", CultureInfo.InvariantCulture);
-                            expressResponse = ppAPI.SetExpressCheckout(
-								itemsDetails,
-								itemsTotal,
-								taxTotal,
-								shippingTotal,
-								orderTotal,
-                                cartReturnUrl,
-                                cartCancelUrl,
-                                mode,
-                                PayPalAPI.GetCurrencyCodeType(context.HccApp.CurrentStore.Settings.PayPal.Currency),
-								solutionType,
-                                address.FirstName + " " + address.LastName,
-                                address.CountryData.IsoCode,
-                                address.Line1,
-                                address.Line2,
-                                address.City,
-                                address.RegionBvin,
-                                address.PostalCode,
-                                address.Phone,
-                                context.Order.OrderNumber + Guid.NewGuid().ToString(),
-								isNonShipping);
+                            expressResponse = System.Threading.Tasks.Task.Run(() => replacePayPal.createOrder(
+                                                    itemsDetails,
+                                                    itemsTotal,
+                                                    taxTotal,
+                                                    shippingTotal,
+                                                    orderTotal,
+                                                    cartReturnUrl,
+                                                    cartCancelUrl,
+                                                    mode,
+                                                    context.HccApp.CurrentStore.Settings.PayPal.Currency,
+                                                    solutionType,
+                                                    address.FirstName + " " + address.LastName,
```

Comment:
Please use interpolation to replace the string using formatting (`string.Format()` or `$""`), and make static strings like this a `constant`.